pkg/core/http.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2022 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package core
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/go-logr/zapr"
+	"go.uber.org/zap"
+
+	"github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/tproxyconfig"
+	"github.com/pingcap/errors"
+)
+
+const (
+	TargetRequest  tproxyconfig.PodHttpChaosTarget = "Request"
+	TargetResponse tproxyconfig.PodHttpChaosTarget = "Response"
+)
+
+const (
+	HTTPAbortAction = "abort"
+	HTTPDelayAction = "delay"
+	HTTPFileAction  = "file"
```

Comment:
I think it's better to change `file` to `config`

Reply:
OK

Reply:
Fixed.